src/app/venter/complaints/complaints-home.component.css

```diff
@@ -0,0 +1,41 @@
+.container {
+    margin: 0 175px;
+}
+
+.toolbar {
+    padding: 150px 0;
+    background-color: #536dfe;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/wncc/instiapp-angular/pullRequest?prid=5455571)